cfe_internal/enterprise/ha/ha_update.cf

```diff
@@ -0,0 +1,141 @@
+bundle agent ha_update
+{
+ vars:
+      "master_hub_location" string => "/var/cfengine/state/master_hub.dat";
+      "hubs_keys_location" string => "/var/cfengine/ppkeys_hubs";
+      "ppkeys_staging" string => "/var/cfengine/ppkeys_staging";
```

Comment:
These paths are also shared by the hub and we have access promises for them right?
If these variables are defined in a common bundle, then the both the access promises in the server bundle and the promises in the agent bundle would be able to reference them. If there was ever a change, we would only have to update one location.